iOS/RootVC.swift

```diff
@@ -78,7 +78,14 @@ public class RootVC: UIViewController {
 
     func toggleAR() {
         if mode == .disabled {
-            mode = .searching
+            if AVCaptureDevice.authorizationStatus(forMediaType: AVMediaTypeVideo) == .notDetermined {
+                AVCaptureDevice.requestAccess(forMediaType: AVMediaTypeVideo) { granted in
```

Comment:
I think that the ARSession will request camera access automatically in the `.notDetermined` case.